### ISSUE TITLE
Bower install reports warning: invalid-meta The "main" field cannot contain minified files

### ISSUE THREAD

**Issue by jdufresne** [TargetProcess/tauCharts#383]
bower.json

```json
{
    "name": "tau-test",
    "dependencies": {
        "taucharts": "*"
    }
}
```

install

```
$ bower install
bower taucharts#*           not-cached https://github.com/TargetProcess/tauCharts.git#*
bower taucharts#*              resolve https://github.com/TargetProcess/tauCharts.git#*
bower taucharts#*             download https://github.com/TargetProcess/tauCharts/archive/1.1.2.tar.gz
bower taucharts#*              extract archive.tar.gz
bower taucharts#*         invalid-meta for:/tmp/jon/bower/cf4727703f5c1a15c2a8f040da4c6572-18234-FyTLRR/bower.json
bower taucharts#*         invalid-meta The "main" field cannot contain minified files
bower taucharts#*         invalid-meta The "main" field cannot contain minified files
bower taucharts#*             resolved https://github.com/TargetProcess/tauCharts.git#1.1.2
bower d3#~3.5.17            not-cached https://github.com/mbostock-bower/d3-bower.git#~3.5.17
bower d3#~3.5.17               resolve https://github.com/mbostock-bower/d3-bower.git#~3.5.17
bower d3#~3.5.17              download https://github.com/mbostock-bower/d3-bower/archive/v3.5.17.tar.gz
bower d3#~3.5.17               extract archive.tar.gz
bower d3#~3.5.17              resolved https://github.com/mbostock-bower/d3-bower.git#3.5.17
bower taucharts#*              install taucharts#1.1.2
bower d3#~3.5.17               install d3#3.5.17

taucharts#1.1.2 bower_components/taucharts
└── d3#3.5.17

d3#3.5.17 bower_components/d3
```

Notice the lines:

```
bower taucharts#*         invalid-meta for:/tmp/jon/bower/cf4727703f5c1a15c2a8f040da4c6572-18234-FyTLRR/bower.json
bower taucharts#*         invalid-meta The "main" field cannot contain minified files
bower taucharts#*         invalid-meta The "main" field cannot contain minified files
```

**Comment by alexanderby** [comment]
Try to use `taucharts@2.0.0-beta.7`, API changes are not documented yet, some short list:
- D3 v4 is now dependency.
- `Taucharts` global object name (was `tauCharts`).
- `exportTo` plugin renamed to `export-to`.
- Dist files names changed.

I haven't checked myself how it works with Bower. What I see from your log is that it tries to use Git repository, but we do not store dist files in Git (we used to do it in `release` branch for v1). Maybe there is a way to install bower dependency from NPM repository or CDN https://cdn.jsdelivr.net/npm/taucharts@2.0.0-beta.7/dist/

Also you should know that minified files contain all the plugins, unminified files do not include plugins.

**Comment by Mavrin** [comment]
Look like it was fixed for taucharts@2.4.0
